Src/AutoMoq/MockVirtualMethodsCommand.cs

```diff
@@ -115,6 +115,13 @@ private static IEnumerable<MethodInfo> GetConfigurableMethods(Type type)
                               ? type.GetInterfaceMethods()
                               : type.GetMethods();
 
+            // Don't mock "GetSet" properties because they will be stubbed.
```

Comment:
Could you please also extend the class documentation (on top of the file) to specify that it excludes the properties that has both setters and getters.

Also the current description doesn't look too much informative. I'd suggest to alter it to smth like "Skip properties that have both getters and setters to not interfere with other post-processors in the chain which initialize them."
or something better :)